### PR TITLE
Homepage

### LINKED ISSUE #11: Home page
## Acceptance criteria : 
- [ ] view all the available events and courses.
- [ ] section where a user can book a room.
- [ ] details button for each event and course.

### PR DESCRIPTION
**Pull request by MohammedYehia**
Fixes
fixes #11 
Related
related to #11 

Summary
adding the home page, courses/events pages and some helpers to customize the date 
restricting the number of the returned data in the home page

Major Changes
homepage [57fbdc7](https://github.com/FACG4/gmpLibrary/pull/15/commits/57fbdc7bc91402d6a235004013bf4d843a88188b)

### CODE REVIEW

File: src/database/models/event_courses.js [hunk 1]
```diff
@@ -3,6 +3,9 @@ export default (sequelize, DataTypes) => {
     title: {
       type: DataTypes.STRING,
     },
+    image: {
```
Comment (mohanagy) [inline src/database/models/event_courses.js]: why image is text 
is this just a url ?

File: package.json [hunk 2]
```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "start": "nodemon --exec babel-node src/index.js",
-    "data": "nodemon --exec babel-node src/database/test_data/test_data1.js"
+    "data": "nodemon --exec babel-node src/database/test_data/test_data3.js"
   },
```
Comment (mohanagy) [inline package.json]: can we have a command for create and drop the database

File: src/database/models/event_courses.js (hunk 1 above)
Comment (MohammedYehia) [reply src/database/models/event_courses.js]: yes its uploaded on cloudinary

File: package.json (hunk 2 above)
Comment (MohammedYehia) [reply package.json]: we will with the test

**Comment by MohammedYehia** [comment]
config file on gitter

File: src/database/models/event_courses.js (hunk 1 above)
Comment (mohanagy) [reply src/database/models/event_courses.js]: so why the datatype is text?

File: package.json [hunk 3]
```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "start": "nodemon --exec babel-node src/index.js",
-    "data": "nodemon --exec babel-node src/database/test_data/test_data1.js"
+    "data": "nodemon --exec babel-node src/database/test_data/test_data3.js"
```
Comment (mohanagy) [inline package.json]: why we need nodemon here?

Comment (mohanagy) [reply package.json]: test_data3 will not work without data2 and 1 
you should handle it in the right way

File: src/views/home.hbs [hunk 4]
```diff
@@ -1 +1,51 @@
...
+    <section class="fieldset">
+      <h2>الفعاليات</h2>
+      <div class="events">
+        {{#each events}}
+      <div class="event">
+        <h2>{{this.title}}</h2>
+        <img class="eventimg" src="{{this.image}}" alt="">
+        <div class="details">{{this.description}}</div>
+        <div class="time">
+          <span>التاريخ:&nbsp;</span><span>{{this.created_at}}</span>
```
Comment (mohanagy) [inline src/views/home.hbs]: is this the date of event?

File: src/database/models/event_courses.js (hunk 1 above)
Comment (MohammedYehia) [reply src/database/models/event_courses.js]: ok now I see  I wil edit it

File: package.json (hunk 3 above)
Comment (MohammedYehia) [reply package.json]: babel wont run without nodemon

File: src/views/home.hbs (hunk 4 above)
Comment (MohammedYehia) [reply src/views/home.hbs]: yes

**Pull request merged**